rllib/agents/ppo/ddppo.py

```diff
@@ -97,6 +97,21 @@
 
 
 class DDPPOTrainer(PPOTrainer):
+    def __init__(self, config, *args, **kwargs):
```

Comment:
can you pass the args and kwargs by name of the DDPPO trainer (you can copy them from the ppo trainer __init__), instead of passing just `*args`, and `**kwargs`. Otherwise this looks good to me

Reply:
No prob. Getting it from Trainer since PPOTrainer doesn't override.